package.json

```diff
@@ -21,6 +21,7 @@
   "repository": "jsdom/jsdom",
   "dependencies": {
     "abab": "^2.0.5",
+    "abort-controller": "^3.0.0",
```

Comment:
This looks like just a `devDependency` since I only see it used in `test/util.js`

Reply:
I see it used in resource-loader

Reply:
Indeed - I missed that. Sorry.